elyra/pipeline/processor_airflow.py

```diff
@@ -96,8 +96,8 @@ def process(self, pipeline):
                                    f"pipeline pushed to git: {github_url}",
                                    duration=(time.time() - t0_all))
 
-            return PipelineProcessorResponse(
-                # TODO - Add another field to return the url of the job in Airflow UI
+            return self.AirflowPipelineProcessorResponse(
```

Comment:
You shouldn't need `self.`
```suggestion
            return AirflowPipelineProcessorResponse(
```